### DISCUSSION TITLE
Alternative Backends

### DISCUSSION THREAD

**Discussion by Wulfheart** [livewire/livewire#4597] [4. General]
If I recall correctly @calebporzio mentioned in his podcast that Livewire will one day become a spec with different backends.
Are there plans to realise this yet?

**Comment by joshhanley** [comment]
@Wulfheart not at this stage, but V3 should get us closer to this possibility 🤞 do you have something in particular you want to use it for?

**Reply by Wulfheart** [reply]
Thanks for your response. 
I would like to port it to .NET. 

Any ETA for v3?

**Reply by joshhanley** [reply]
Ah yeah fair enough. No ETA yet.
